jfr-mappers/src/main/java/com/newrelic/jfr/profiler/StackTraceEvent.java

```diff
@@ -0,0 +1,3 @@
+package com.newrelic.jfr.profiler;
+
+public interface StackTraceEvent {}
```

Comment:
Not quite sure I see the point of this marker interface.

Reply:
good point, removed